src/reducers/entities/posts.js

```diff
@@ -384,6 +411,8 @@ function handlePosts(posts = {}, postsInChannel = {}, postsInThread = {}, action
         return {posts, postsInChannel, postsInThread};
     case PostTypes.REMOVE_POST:
         return handleRemovePost(posts, postsInChannel, postsInThread, action);
+    case PostTypes.RETHREAD_POST_UPDATE:
```

Comment:
This isn't the action you've called attached to the `rethreadPost` action above, so this code is never actually ran